test/inductor/test_gpu_cpp_wrapper.py

```diff
@@ -23,6 +22,7 @@
         )
     except ImportError:
         import test_combo_kernels  # @manual=fbcode//caffe2/test/inductor:combo_kernels-library
+        import test_cpu_cpp_wrapper  # @manual=fbcode//caffe2/test/inductor:cpu_cpp_wrapper-library
```

Comment:
I don't know if this is the correct path. I've deleted make_test_case from this module because its the same as in test_cpu_cpp_wrapper